src/main/org/nlogo/headless/ChecksumsAndPreviews.scala

```diff
@@ -41,6 +41,8 @@ object ChecksumsAndPreviews {
   }
 
   object Previews {
+    def needsManualPreview(previewCommands: String) =
+      previewCommands contains "need-to-manually-make-preview-for-this-model"
```

Comment:
This code will hopefully be short-lived anyway: I have something else in store for `6.x` that should look much better to Scala eyes... :-)